### PR TITLE
Update libbeat Path to elastic/libbeat

### PR DESCRIPTION
**Pull request by oren-zohar**
relevant: https://github.com/elastic/security-team/issues/3191

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -127,7 +126,6 @@ require (
 	github.com/eapache/go-resiliency v1.2.0 // indirect
 	github.com/elastic/elastic-agent-client/v7 v7.0.0-20210727140539-f0905d9377f6 // indirect
 	github.com/elastic/go-concert v0.2.0 // indirect
-	github.com/elastic/go-licenser v0.4.0 // indirect
```
Comment (ofiriro3) [inline go.mod:130]: I am curious about this one, probably they change the way they check the license

File: go.sum [hunk 2]
```diff
@@ -477,7 +476,8 @@ github.com/eapache/queue v1.1.0/go.mod h1:6eCeP0CKFpHLu8blIFXhExK/dRa7WDZfr6jVFP
 github.com/eclipse/paho.mqtt.golang v1.2.0/go.mod h1:H9keYFcgq3Qr5OUJm/JZI/i6U7joQ8SYLhZwfeOo6Ts=
 github.com/eclipse/paho.mqtt.golang v1.3.5/go.mod h1:eTzb4gxwwyWpqBUHGQZ4ABAV7+Jgm1PklsYT/eo8Hcc=
 github.com/edsrzf/mmap-go v1.0.0/go.mod h1:YO35OhQPt3KJa3ryjFM5Bs14WD66h8eGKpfaBNrHW5M=
-github.com/elastic/csp-security-policies v0.0.2-go-lib/go.mod h1:24NNr0b/5HTGtndJOmhrefb59rd7NjuqI/To39tgn+w=
+github.com/elastic/beats/v7 v7.0.0-alpha2.0.20220309143526-12977c7c9ca9 h1:EIJgR5QImXI5xDuQ4D0EvZmsxEakUkTwF/FO8odMKY4=
```
Comment (ofiriro3) [inline go.sum:479]: 👍

**Comment by ofiriro3** [comment]
Looks great - GJ!

Could be great if you can update the documentation\ send a short message over slack regarding how should we update the beats from now on.

File: go.mod (hunk 1 above)
Comment (oren-zohar) [reply go.mod]: nice catch, it was removed by `go mod tidy` since we use it in the mage file and it is ignored by the build

**Review by ofiriro3 — APPROVED** [review]

**Pull request merged**